packages/blocks/src/code-block/code-block.ts

```diff
@@ -199,13 +199,7 @@ export class CodeBlockComponent extends NonShadowLitElement {
   langListElement!: HTMLElement;
 
   @state()
-  showLangList = 'hidden';
-
-  @state()
-  disposeTimer = 0;
-
-  @state()
-  filterText = '';
+  private showLangList = 'hidden';
```

Comment:
Well I think we can enforce the `_showLangList` convention for private variable here 🤔